src/app/tests/suites/certification/Test_TC_TBRM_2_5.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    "[TC-TBRM-2.5] Verify network configuration of a Thread Border Router
+    endpoint that support also Secondary Network Interface device type"
+
+PICS:
+    - TBRM.S
+    - CNET.S.F01
+
+config:
+    nodeId: 0x12344321
+    cluster: Thread Border Router Management
+    endpoint: 1
+    PIXIT.TBRM.THREAD_ACTIVE_DATASET:
+        type: octet_string
+        defaultValue: "hex:0e08000000000001000035060004001fffe00708fddeb3d807f0cc3f04104b575fabba5fdb5917c8267eca0674740c0402a0f7f8000300001901025b3502085b35dead5b35beef030435623335051000112233445566778899aabbccddeeff"
```

Comment:
This is not the usual method to pass dataset for commissioning. This means that the step in the test plan about commissioning over Thread with BLE may not use this dataset, and therefore the dataset checked against may not be the one used during commissioning.

Reply:
The commissioning process is done by the test harness. The test script assume that the commissioning is done already and try to retrieve the commissioned device. User need to configure the correct PIXIT value in the Harness Project configuration, or pass it as an argument to the script if is executed manually and not with the Harness GUI (meaning the one from the OTBR that is used in the commissioning process) before starting test execution.

Reply:
> meaning the one from the OTBR that is used in the commissioning process

The OTBR-based sample dataset is not the only one that can be used/present. Our DUTs use custom dataset.

The default commissioning of the TH will always use the primary network interface.